docs/explanation.rst

```diff
@@ -0,0 +1,86 @@
+***********
+Explanation
+***********
+
+Getting past the technical matters surrounding Rockcraft, from a higher perspective, 
+you might be asking *"but what is this after all?"* and *"why do I need it?"*.
+
+Let's then use this page to go a bit deeper into the concepts and definitions behind Rockcraft.
+
+
+
+What is a ROCK?
+===============
+
+Rockcraft builds ROCKs, but **what is a ROCK**?
+
+In short, a ROCK is just an OCI-compliant container image! Period.
+
+A ROCK can live on any existing container registry, very much like any other Docker image out there. 
+You can run a ROCK very much like any other container image...for example: ``docker run <rock> ...`` will work 
+just fine. 
+
+And the same applies to container image builds, in the sense that if you already have image build recipes (like Dockerfiles) 
+and you want to start basing your own images on an existing ROCK, that will work just fine too!
+
+
+So why do I need ROCKs?
+=======================
+
+Now, this is the things get interesting. To answer this question, we first need to look at the current state of the art with respect to the existing container image offerings
+out there.
+
+It is easy to find public studies (like `Unit 42 / Znet <https://www.zdnet.com/article/96-of-third-party-container-applications-deployed-in-cloud-infrastructure-contain-known-vulnerabilities-unit-42/>`_ 
+and `Snyk's state of open source security report 2020 <https://snyk.io/blog/10-docker-image-security-best-practices/>`_) where the findings
+state a concerning number of containers at risk deployed in cloud infrastructures.
+
+In fact, both these studies and our own assessments (dated from December 2021) show that the most popular images in Docker Hub 
+contain known vulnerabilities, with Ubuntu being the only one without any critical or high ones.
+
+.. image:: _static/container-image-vulnerabilities.png
+  :align: center
+  :width: 60%
+  :alt: Most popular container images contain known vulnerabilities
+
+Sure, consumers could venture to fix these vulnerabilities themselves, but not only this would increase the cost and scattering of images, but it wouldn't be 
```

Comment:
I think I know what you mean with "scattering", but it seems unclear. Is there a more specific synonym?

Reply:
proliferation is a better word :) changing